plugins/modules/network/icx/icx_firmware_upgrade.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+---
+module: icx_firmware_upgrade
+version_added: "1.2.0"
```

Comment:
1.2.0 has already been released. The next release will be 1.3.0 next week so use that one. It needs to be updated to 2.0.0 if the module doesn't get merged by then.